Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -4016,6 +4016,260 @@ public IEnumerator UI_DisplayIndexMatchesDisplayMultiplePointers()
         Assert.AreEqual(0, scene.parentReceiver.events.Count);
     }
 
+    void ClickButton(Button button)
+    {
+        bool called = false;
+        button.onClick.AddListener(() => { called = true; });
+        button.OnPointerClick(new PointerEventData(EventSystem.current) { button = PointerEventData.InputButton.Left });
+        //Debug.Log(called);
+        Assert.AreEqual(called, true);
+    }
+
+    void ChangeCanvasRenderMode(Canvas canvas, RenderMode mode)
+    {
+        canvas.renderMode = mode;
+    }
+
+    [Test]
+    public void ButtonClickTest()
```

Comment:
I think this functionality is already covered by these tests:

[UI_DisplayIndexMatchesDisplayWithTouchscreenOnOverlayCanvas
UI_DisplayIndexMatchesDisplayWithTouchscreenOnScreenSpaceCanvas](https://github.com/Unity-Technologies/InputSystem/pull/1603/commits/6438dde73bbcb06e997a9019e201ff0c69401f9e#diff-21434067f6fafbece8297b35cc73ac692f2add4300e88703810bf8be4556c91bR3767)